src/System.Management/src/Resources/Strings.resx

```diff
@@ -284,4 +284,7 @@
   <data name="PlatformNotSupported_SystemManagement" xml:space="preserve">
     <value>System.Management currently is only supported for Windows desktop applications.</value>
   </data>
+  <data name="PlatformNotSupported_FullFrameworkRequired" xml:space="preserve">
+    <value>The .NET Framework location was not found in the registry. System.Management requires native modules from the .NET Framework to operate.</value>
```

Comment:
Since the 99% case will be it's a OneCore SKU like Nano or IoT, a better message might be something like
```xml
<value>Could not find an installation of .NET Framework v4.0.30319. System.Management requires native modules from the .NET Framework to operate.</value>
```

Reply:
Sounds better indeed